src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -51,6 +68,19 @@ export default (data, service, currentTime) => {
         publishedTimeDuration,
       } = program;
 
+      const brandTitle = path(['brand', 'title'], program);
+      const episodeTitle = path(['episode', 'presentationTitle'], program);
```

Comment:
We now don't need to use episodeTitle at all, since version 2.0.0 of psammead-radio-schedules does not need it. 

https://github.com/bbc/psammead/pull/3377/files#diff-daa9b5cfc0c949bd0a8f361495e8a040L147

We can remove this from here.

Reply:
From the screenshot Laura shared, episodeTitle appears to be a minimum requirement. 
<img width="447" alt="Screen Shot 2020-04-22 at 14 12 16" src="https://user-images.githubusercontent.com/34133081/80005542-a201d000-84bb-11ea-8e98-97dea2c4dd5a.png">